Brewfile

```diff
@@ -39,10 +38,11 @@ brew 'kubernetes-cli'
 brew 'the_silver_searcher'
 brew 'diff-so-fancy'
 
+# These need the icloud?
```

Comment:
Yes, in order to install apps from the Apple App Store you need to have logged in to cloud.